src/modules/reading-room/components/FilterMenu/FilterMenuMobile/FilterMenuMobile.tsx

```diff
@@ -1,29 +1,71 @@
-import { Button } from '@meemoo/react-components';
-import { FC } from 'react';
+import { Button, TagList } from '@meemoo/react-components';
+import clsx from 'clsx';
+import { FC, ReactElement } from 'react';
 
 import { Icon, Navigation } from '@shared/components';
 
+import { FilterMenuFilterOption } from '..';
+
 import styles from './FilterMenuMobile.module.scss';
 import { FilterMenuMobileProps } from './FilterMenuMobile.types';
 
-const FilterMenuMobile: FC<FilterMenuMobileProps> = ({ isOpen, onClose }) => {
+const FilterMenuMobile: FC<FilterMenuMobileProps> = ({
+	activeFilter,
+	isOpen,
+	filters,
+	onClose,
+	onFilterClick = () => null,
+}) => {
 	if (!isOpen) {
 		return null;
 	}
 
+	// TODO: move to separate component
```

Comment:
Moet deze todo nog gedaan worden?

Reply:
Jep, vandaar de todo 😄 komt eraan in een volgende PR.